pyalgotrading/constants.py

```diff
@@ -80,67 +80,67 @@ class CandleInterval(Enum):
     """
     A class of Candle Intervals with their Alternate names
     """
-    MINUTES_1 = 0
```

Comment:
`class AlgoBullsEngineVersion(Enum):`

Version 2 has been phased out, kindly confirm with Pushpak
if confirmed, then that entry can be removed

Reply:
@pushpakdagade Please look into this.

Reply:
removed;